Packs/RiskIQDigitalFootprint/ReleaseNotes/1_0_4.md

```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### RiskIQ Digital Footprint
+- **Breaking Change** df-asset-connections, df-get-asset commands and df-asset-connections changed to return multiple entries (entry per indicator) instead of a single entry.
```

Comment:
```suggestion
- **Breaking Change** the **df-asset-connections**, **df-get-asset commands** and **df-asset-connections changed** commands were changed to return multiple entries (entry per indicator) instead of a single entry.
```